hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e.yaml

```diff
@@ -226,6 +226,24 @@
         - 'kubernetes-e2e-{suffix}'
 
 - project:
+    name: kubernetes-e2e-gce-1-2
+    trigger-job: 'kubernetes-build-1.2'
+    test-owner: 'Build Cop'
+    provider-env: |
+        {gce-provider-env}
+        export JENKINS_PUBLISHED_VERSION="ci/latest-1.2"
```

Comment:
Yes, I want to wait until we've got the 1.2 jobs in then do a big YAML cleanup. We can start doing things like putting `export JENKINS_PUBLISHED_VERSION="{published-version}"` in the build script along with a global default to `ci/latest` and stuff. It'll be nice.